docs/how-to/existingLicense.md

```diff
@@ -90,3 +90,18 @@ pingfederate-admin:
       items:
         pingfederate.lic: /opt/in/instance/server/default/conf/pingfederate.lic
 ```
+
+### Note on updating the product license when mounting it as a file
+If you are updating the license file for a product, simply replacing the file on the filesystem may not update the license of the running server.
+
+For PingData products (PingDirectory, PingDataSync, PingAuthorize, and PingDirectoryProxy) the license can be updated by copying the new license to the expected location in the server profile - `pd.profile/server-root/pre-setup`. Then dsconfig can be used to update the license on the running server. Ensure that the updated license file is still present in the server profile on subsequent restarts of the container.
+
+For example, for PingDirectory:
+```
+dsconfig set-license-prop \
+  --set "directory-platform-license-key<input-file.lic"
+```
+
+The exact name of the license property in the above example will depend on which PingData product is being used.
+
+For non-PingData products, the license can be updated on the product with the typical method. This will depend on the product, but can generally be done either via the administrative console or via an API call. See the product documentation for details.
```

Comment:
...method.  This process will depend on the product, but will generally be done either through the administrative console or using an API call.  ...